editor/assets/stylesheets/_z-index.scss

```diff
@@ -5,6 +5,7 @@
 $z-layers: (
 	'.editor-block-switcher__arrow': 1,
 	'.editor-visual-editor__block:before': -1,
+	'.editor-visual-editor__block .wp-block-more before': -1,
```

Comment:
I guess it doesn't matter much, but other keys leave the `:` (colon) of `:before`